src/main/java/edu/harvard/iq/dataverse/TemplatePage.java

```diff
@@ -133,7 +130,7 @@ public String init() {
             } else {
                 TermsOfUseAndAccess terms = new TermsOfUseAndAccess();
                 terms.setTemplate(template);
-                terms.setLicense(TermsOfUseAndAccess.License.CC0);
+                terms.setLicense(licenseServiceBean.getDefault());
```

Comment:
@qqmyers after assessing the code again and refreshing my memory, I realized that in this `multi-license` feature, setting TermsOfUseAndAccess, indirectly means using a Custom License (meaning License is set to `null`) and when a License is set TermsOfUseAndAccess is set to `null`. That means I shouldn't have been setting the default license here but setting it to `null` instead. What I assumed here was to use the old license behavior. Correct me if i'm wrong @janvanmansum

Reply:
Correction, setting the default license in the constructor of TermsOfUseAndAcces is the logical choice.

Reply:
Or it seemed like the logical choice... But I'd have to use LicenseServiceBean in an Entity class to fetch the default license from the database. I don't think that is supposed to be done there... I could be missing a different method to get the default value that is stored in the database, but it seems that the construction above might have been used because of this?